README.md

```diff
@@ -283,19 +312,19 @@ python cert-issuer -c conf.ini
 ```
 
 3. Output
-  - The Blockchain Certificates will be located in data/blockchain_certificates.
-  - If you ran in the mainnet or testnet mode, you can also see your transaction on a live blockchain explorer.
-    - For Bitcoin, Blockr.io has explorers for both [testnet](https://tbtc.blockr.io/) and [mainnet](https://blockr.io/).
-    - For Ethereum, Etherscan has explorers for [ropsten](https://ropsten.etherscan.io/) and [mainnet](https://etherscan.io/)
-    - The transaction id is located in the Blockchain Certificate under `signature.anchors[0].sourceId`
 
+- The Blockchain Certificates will be located in data/blockchain_certificates.
+- If you ran in the mainnet or testnet mode, you can also see your transaction on a live blockchain explorer.
+  - For Bitcoin, Blockr.io has explorers for both [testnet](https://tbtc.blockr.io/) and [mainnet](https://blockr.io/).
+  - For Ethereum, Etherscan has explorers for [ropsten](https://ropsten.etherscan.io/) and [mainnet](https://etherscan.io/)
+  - The transaction id is located in the Blockchain Certificate under `signature.anchors[0].sourceId`
```

Comment:
This has likely changed with MerkleProof2019

Reply:
I haven't looked into this part yet.

Reply:
no worries, that's a bit out of scope with what you are doing, it was more a mental note for me